library/ecp.c

```diff
@@ -43,6 +43,10 @@
 
 #include "common.h"
 
+#include "ecp_invasive.h"
+
+#if defined(MBEDTLS_ECP_WITH_MPI_STRUCT)
```

Comment:
why not get rid of `MBEDTLS_ECP_WITH_MPI_STRUCT` and just do

```
#if !defined(MBEDTLS_ECP_WITH_MPI_UINT)
```

here?

We have a boolean condition ("use new code/use old code") that is now tracked with two different variables, that need to remain synchronised